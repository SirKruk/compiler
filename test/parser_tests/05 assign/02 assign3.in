x = y * z / 13 + 14.5
