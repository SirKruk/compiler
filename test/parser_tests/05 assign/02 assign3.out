<ASSIGN
  <ID x>
  <ADD
    <DIV
      <MUL
        <ID y>
        <ID z>
      >
      <INT 13>
    >
    <DOUBLE 14.5>
  >
>
